.github/workflows/ci-main.yaml

```diff
@@ -12,12 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: "Continuous Integration - Master/Release"
+name: "Continuous Integration - Main/Release"
```

Comment:
Since the Action is changing, we'd need to also modify the URL for the CI badge in the root `README.md` from
`![Continuous Integration](https://github.com/GoogleCloudPlatform/bank-of-anthos/workflows/Continuous%20Integration%20-%20Master/Release/badge.svg)`
to
`![Continuous Integration](https://github.com/GoogleCloudPlatform/bank-of-anthos/workflows/Continuous%20Integration%20-%20Main/Release/badge.svg)`